php/ext/google/protobuf/protobuf.h

```diff
@@ -817,6 +821,7 @@ void internal_add_generated_file(const char* data, PHP_PROTO_SIZE data_len,
                                  bool use_nested_submsg TSRMLS_DC);
 void init_generated_pool_once(TSRMLS_D);
 void add_handlers_for_message(const void* closure, upb_handlers* h);
+void register_class(void *desc, bool is_enum TSRMLS_DC);
```

Comment:
So is `register_class` a function that already existed before this PR?

Reply:
It existed as a static method in def.c.
In this change, I refactored a little to make it easy to use.